components/Alert/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { InfoIcon } from '@chakra-ui/icons';
+import { Container, Fade, Text } from '@chakra-ui/react';
+
+interface AlterProps {
+  text: string;
+}
+
+export default function Alert({ text }: AlterProps) {
```

Comment:
Good start on this. Looking at this design [here](https://www.figma.com/file/uFHKfLFYczHnZ7voSluRvO/Chakra-UI-SNX-System?node-id=517%3A9808), it looks like we have primary, secondary, tertiary variants and success, info, warning and error states.

Reply:
Also haven't had a convo with @bachstatter or @noisekit about it, but it would be good by default if we have the option to pass a style object to all our primitives

Reply:
Ah didn't knew that this file existed! I was doing the component based of the burn flow in a different file. Well there is plenty to add here. Will do, thanks!